worksheet_A/TerminalHacking.cpp

```diff
@@ -3,10 +3,42 @@
 
 #include "stdafx.h"
 #include "WordList.h"
+#include <string>
+#include <iostream>
```

Comment:
It's better to add these to `stdafx.h` -- take advantage of precompiled headers to make recompilation faster